src/Calculator/Calculator.vcxproj

```diff
@@ -17,7 +17,7 @@
     <AppxDefaultResourceQualifierUAP_Contrast>black</AppxDefaultResourceQualifierUAP_Contrast>
     <AppxBundle>Always</AppxBundle>
     <PackageCertificateKeyFile>TemporaryKey.pfx</PackageCertificateKeyFile>
-    <AppxPackageSigningEnabled>true</AppxPackageSigningEnabled>
+    <AppxPackageSigningEnabled>True</AppxPackageSigningEnabled>
```

Comment:
revert this

Reply:
I would recommend to not revert it. "true" was a bug, VS will always try to replace it with "True" when you edit the vcproj